### PR TITLE
fix return type at evaluate function

### PR DESCRIPTION
**Pull request by Spyros-Av**

### CODE REVIEW

File: poker.js [hunk 1]
```diff
@@ -87,49 +87,76 @@ for (let i = 0; i < sorted.length - 1; i++) {
...
+  this.hand = handType;
+  this.highCard = strength;
+}
+
 function evaluate(hand) {
-  if (suitHand.find(num => num === 5) && isSorted.every(num => num === -1) && isSorted.length === 4) {
+  const strength = highCard(rank);
+  if (suitHand.find(num => num === 5) && isSorted.every(num => num === -1) && isSorted.length === 4 && strength === 14) {
+    console.log('Your hand is Royal Flush');
+    const player = new PlayerHand(handType.ROYAL_FLUSH, strength);
```
Comment (stacon) [inline poker.js]: Keep your naming consistent even if it is verbose `playerHand` is more descriptive. What do you think

File: poker.js [hunk 2]
```diff
@@ -87,49 +87,76 @@ for (let i = 0; i < sorted.length - 1; i++) {
...
+    highestCard < elem ? highestCard = Number(elem) : highestCard;
   }
   return highestCard;
 };
 
+const PlayerHand = function (handType, strength) {
+  this.hand = handType;
+  this.highCard = strength;
+}
+
```
Comment (stacon) [inline poker.js]: Lets go with a more functional approach I'd say..
`new` keyword isn't needed anymore
```suggestion
const PlayerHand = (hand, highCard) => ({
  hand,
  highCard,
});

```

File: poker.js [hunk 3]
```diff
@@ -56,24 +55,25 @@ const getValues = (groupedHand, maped) => {
 const suit = hand.map(item => item.suit);
 // console.log('----Suit----')
 // console.log(suit);
+
 const rank = hand.map(item => item.rank);
 for (let i = 0; i < rank.length; i++) {
   (rank[i] === 'J' || rank[i] === 'j') ? rank[i] = 11 :
```
Comment (stacon) [inline poker.js]: You can always lowercase the rank[i] before doing the comparison just for shorter and cleaner code.

**Comment by stacon** [comment]
@Spyros-Av I don't see the requested fixes from the code review on the pull request. Do you need some help with that?

**Comment by Spyros-Av** [comment]
> @Spyros-Av I don't see the requested fixes from the code review on the pull request. Do you need some help with that?

Iadded an object constructor from line 95 to 98, also the 'evaluate' function return objects.  Aren't these changes visible?

**Comment by stacon** [comment]
I still see the old codebase @Spyros-Av 
image

We can catchup tomorrow to give you a small tour on GitHub usage. What do you think?

**Comment by Spyros-Av** [comment]
> I still see the old codebase @Spyros-Av
> image
> 
> We can catchup tomorrow to give you a small tour on GitHub usage. What do you think?

Sure. Lets catch up tomorrow, I believe it's a good idea.

**Review by stacon — CHANGES_REQUESTED** [review]
Check comments

File: poker.js (hunk 2 above)
Comment (Spyros-Av) [reply poker.js]: Done!

**Review by stacon — APPROVED** [review]
Good job

**Pull request merged**